0-bootstrap/main.tf

```diff
@@ -101,8 +100,7 @@ resource "google_billing_account_iam_member" "tf_billing_admin" {
 
 // Comment-out the cloudbuild_bootstrap module and its outputs if you want to use Jenkins instead of Cloud Build
 module "cloudbuild_bootstrap" {
-  source                      = "terraform-google-modules/bootstrap/google//modules/cloudbuild"
-  version                     = "~> 5.0"
+  source                      = "github.com/terraform-google-modules/terraform-google-bootstrap//modules/cloudbuild"
```

Comment:
I just a cut a 6.0 release so we can switch this back to the registry and `"~> 6.0"`

Reply:
Done, switch it back to 6.0

Reply:
Hi @iyabchen @bharathkkb 
I tested all stages with  **terraform** `0.13.7` and `1.1.9` .

we have two issues:

1)  in step 3-networks we have this old issue not directly related to  `gcloud beta terraform vet` but that fails the build:
https://github.com/GoogleCloudPlatform/policy-library/issues/414

2) in steps `0-bootstrap`, due to  `google_folder_iam_member` and `google_folder_iam_binding`, and in step `4-projects` due to `google_folder_iam_member` we have this crash:
https://github.com/GoogleCloudPlatform/terraform-validator/issues/708

I removed the usage of `google_folder_iam_member` and `google_folder_iam_binding` from steps 0 and 4 and there is no other issues related to `gcloud beta terraform vet` on those steps.

the [crash has been fixed](https://github.com/GoogleCloudPlatform/terraform-validator/pull/764) but it in not yet in a [gcloud release](https://cloud.google.com/sdk/docs/release-notes)

I can retest the json files created for the stages after a new `gcloud` release that contains the fix

Reply:
@daniel-cit , gcloud cli 393.0.0 is released, and it is with terraform-tools 0.5.0, which should cover the fix for [crash issue](https://github.com/GoogleCloudPlatform/terraform-validator/pull/764) you mentioned. 
Would you be possible to do a retest?

Reply:
Hi @iyabchen I will do the retest

Reply:
foundation deploy retested with `gcloud cli 393.0.0` and all steps were successful 🎉 

```
Validating resources....done.
```

PR will need to be updated when  393.0.0-slim is available at [Docker Hub](https://hub.docker.com/r/google/cloud-sdk/tags)

Reply:
@daniel-cit I updated to 393.0.0, and the tests are failing for 409 errors... I don't quite get it. 
If the bootstrap repo also need to update to 393.0.0 and trigger a new release, let me know.

Reply:
the `Error 409: Requested entity already exists` is a name collision with a deleted project that is still in the 30 days interval  in which it can be restored.

I rerun the builds and I will keep track of ti

Reply:
Seems the test run into something new `Error deleting folder 'fldr-production': googleapi: Error 400: Failed to delete folder [folders/38748887386] due to: The folder being deleted is non-empty.`

Reply:
It failed differently on rerun: `Error enabling the Compute Engine API required to delete the default network`

Reply:
all checks passed